### PR TITLE
feat(apps-web): preview for nodes, tree view

### PR DESCRIPTION
**Pull request by aburtsev**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by webberwang — CHANGES_REQUESTED** [review]
A general pattern I noticed is using conditional statements when not required

**Review by aburtsev — COMMENTED** [review]
I've fixed some of your remarks, but somewhere just left a comment. Please have a look. (I've rebased all commit into one 'code-review' for your convenience but merged into this commit fixes after merging master. So there will be as fixes for code review  as well as fixes after merge master)

File: apps/web/pages/node.tsx [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
 axios.defaults.headers.post['Content-Type'] = 'application/json'
 
 const NodePage = () => {
-  const [selectedNode, setSelectedNode] = React.useState(null)
-  const [rootNode, setRootNode] = React.useState<Node | null>(null)
-  const [treeDataNodes, setTreeDataNodes] = React.useState<Array<DataNode>>([])
+  const [selectedNode, setSelectedNode] = React.useState<NodeEntity | null>(
+    null,
+  )
+  const [rootNode, setRootNode] = React.useState<NodeEntity | null>(null)
```
[the remark this answers is not in the document]
Comment (webberwang) [reply apps/web/pages/node.tsx:25]: Entry point can be any node, since they are all connected

**Comment by webberwang** [comment]
@aburtsev hey, could you clone master and move your old unmerged module to pages/tree instead.
 
I think it’s better to keep them separate for now

**Comment by webberwang** [comment]
Would this be closed by https://github.com/codelab-ai/codelab.ai/pull/165?

**Comment by aburtsev** [comment]
> Would this be closed by #165?

Yes. I've moved all changes into the "tree" page

**Pull request closed without merging**